### ISSUE TITLE
Unknown prop warning

### ISSUE THREAD

**Issue by jamieshepherd** [eiriklv/react-masonry-component#44]
Updated React today to be greeted with this warning in the console. Haven't changed anything from my masonry component, still trying to figure out exactly how to prevent it. Only Masonry triggering this warning. My code is standard from README.md.

![image](https://cloud.githubusercontent.com/assets/5076961/16587140/15ac8e5a-42c1-11e6-866e-7773cac7606d.png)

Linked: https://fb.me/react-unknown-prop

**Comment by jamieshepherd** [comment]
````<Masonry
                    className={'my-gallery-class'}
                    options={this.masonryOptions}
                    disableImagesLoaded={false}>
                    { childElements }
                </Masonry>

**Comment by afram** [comment]
Thanks for reporting @jamieshepherd will investigate tonight

**Comment by afram** [comment]
@jamieshepherd this is happening because those props are being passed down to the div this component creates.

I'll update the code base and will let you know when it's released

**Comment by emzero** [comment]
I was about to post the same issue. 
The custom props don't have to be passed down to the rendered div.

**Comment by afram** [comment]
fixed in v4.2.0
